pyproject.toml

```diff
@@ -46,5 +46,5 @@ build-backend = "poetry.core.masonry.api"
 profile = "black"
 
 [tool.pytest.ini_options]
-markers = ["unit", "integration", "smoke", "slow"]
+markers = ["unit", "integration", "smoke", "slow", "wip"]
```

Comment:
proposal for enhancement ticket: since we can call markers with

```
@pytest.mark.wip("2023-11-23")
```

we can effectively set an expiration date on `wip` markers so that they don't persist forever, or at least we are reminded to implement them or remove them